sunpy/io/cdf.py

```diff
@@ -71,18 +71,28 @@ def read_cdf(fname):
 
             data = cdf.varget(var_key)
             # Get units
-            unit_str = attrs['UNITS']
             try:
```

Comment:
Instead of catching a `KeyError` here, I think it would be better to read with `if 'UNITS' in attrs:` instead.

Reply:
Yes, that's actually more elegant. Don't know why I took a more complicated way. 😬